<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preprocessor</class>
 <widget class="QDialog" name="preprocessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SUPR препроцессор</string>
  </property>
  <widget class="QTableWidget" name="table_kernel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>921</width>
     <height>201</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>i- стержень</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>L-длинна</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>A-поперечное сечение</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>E-модуль упругости</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> [σ] допускаемое напряжение</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="add_line">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>240</y>
     <width>133</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>добавить строку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delite_line">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>240</y>
     <width>124</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>удалить строку</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table__load">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>q-Статическая погонная нагрузка</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="PB_close">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
